### PR TITLE
First draft of the adb summary

### PR DESCRIPTION
**Pull request by rylyade1**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1228 .

### CODE REVIEW

File: Document/0x05b-Basic-Security_Testing.md [hunk 1]
```diff
@@ -107,12 +107,20 @@ For a typical mobile app security build, you'll usually want to test a debug bui
 
 -- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1228>
 
-[adb](https://developer.android.com/studio/command-line/adb "Android Debug Bridge") (Android Debug Bridge) ships with the Android SDK, bridges the gap between your local development environment and a connected Android device. You'll usually debug apps on the emulator or a device connected via USB. Use the `adb devices` command to list the connected devices.
+[adb](https://developer.android.com/studio/command-line/adb "Android Debug Bridge") (Android Debug Bridge) ships with the Android SDK, bridges the gap between your local development environment and a connected Android device. You'll usually leverage it to test apps on the emulator or a connected device via the USB or WiFi. Use the `adb devices` command to list the connected devices and execute it with the `-l` argument to retrieve more details on these devices.
```
Comment (sushi2k) [inline Document/0x05b-Basic-Security_Testing.md]: via the USB or WiFi -> via USB or WiFi

File: Document/0x05b-Basic-Security_Testing.md [hunk 2]
```diff
@@ -107,12 +107,20 @@ For a typical mobile app security build, you'll usually want to test a debug bui
...
+090c285c0b97f748 device
+emulator-5554    device
+```
+
+```shell
+$ adb devices -l
+List of devices attached
+090c285c0b97f748 device usb:1-1 product:razor model:Nexus_7 device:flo
+emulator-5554    device product:sdk_google_phone_x86 model:Android_SDK_built_for_x86 device:generic_x86 transport_id:1
 ```
```
Comment (sushi2k) [inline Document/0x05b-Basic-Security_Testing.md]: Please also add at least "adb shell" which allows you to get a shell from the android device/simulator and explain that many other things can be done with it, like copying files, port forwarding etc which will be explained in differenct sections in this chapter

File: Document/0x05b-Basic-Security_Testing.md [hunk 3]
```diff
@@ -107,12 +107,20 @@ For a typical mobile app security build, you'll usually want to test a debug bui
...
 
-[adb](https://developer.android.com/studio/command-line/adb "Android Debug Bridge") (Android Debug Bridge) ships with the Android SDK, bridges the gap between your local development environment and a connected Android device. You'll usually debug apps on the emulator or a device connected via USB. Use the `adb devices` command to list the connected devices.
+[adb](https://developer.android.com/studio/command-line/adb "Android Debug Bridge") (Android Debug Bridge) ships with the Android SDK, bridges the gap between your local development environment and a connected Android device. You'll usually leverage it to test apps on the emulator or a connected device via the USB or WiFi. Use the `adb devices` command to list the connected devices and execute it with the `-l` argument to retrieve more details on these devices.
 
 ```shell
 $ adb devices
 List of devices attached
-090c285c0b97f748  device
+090c285c0b97f748 device
+emulator-5554    device
```
Comment (sushi2k) [inline Document/0x05b-Basic-Security_Testing.md]: You can remove this, doing 'adb devices -l' should be sufficient.

File: Document/0x05b-Basic-Security_Testing.md (hunk 2 above)
Comment (rylyade1) [reply Document/0x05b-Basic-Security_Testing.md]: I have added a write up to "adb shell" and "adb forward".

**Pull request merged**